src/mlpack/methods/preprocess/load_save_image_main.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * @file load_save_image_main.cpp
+ * @author Jeffin Sam
+ *
+ * A CLI executable to load and save a image dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core.hpp>
+
+using namespace mlpack;
+using namespace mlpack::util;
+using namespace arma;
+using namespace std;
+using namespace mlpack::data;
+
+
+PROGRAM_INFO("Load Save Image",
+    // Short description.
+    "A utility to load and save image dataset. This utility will allow you to "
+    "load and save a single image or an array of images.",
```

Comment:
Do you think, we should use ``list`` instead of ``array`` here?

Reply:
Yes